src/app/app.module.ts

```diff
@@ -68,21 +69,14 @@ import { LinkOrganizationComponent } from './shared/components/link-organization
 import { LinkOrganizationTableRowsComponent } from './shared/components/link-organization/link-organization-table-rows/link-organization-table-rows.component';
 import { ContactSelectComponent } from './shared/components/contact-select/contact-select.component';
 import { ContactSelectTableRowsComponent } from './shared/components/contact-select/contact-select-table-rows/contact-select-table-rows.component';
+import { FileUploadModalComponent } from './file-upload-modal/file-upload-modal.component';
 
 export function kcFactory(keycloakService: KeycloakService) {
   return () => keycloakService.init();
 }
 
 @NgModule({
   declarations: [
-    // ProjectUpdatesComponent,
```

Comment:
Were these declarations moved elsewhere (like in Projects declarations)? It looks like they were commented before, but these are definitely pretty important to the functionality.

Reply:
I believe these were all moved to the Project module. This goes for the "Activity" components since we stopped treating them as a standalone thing and started associating them with particular projects.

These have been commented out since the time of that work which was almost a year ago, so we should be good